### PR TITLE
Reorg

### PR DESCRIPTION
**Pull request by catlee**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,8 +1,5 @@
 recursive-include requirements *
-recursive-include signingscript/test *
-recursive-include signingscript/data *
-
-recursive-include signingscript/vendored *
+recursive-include src/signingscript
```
Comment (escapewindow) [inline MANIFEST.in]: If you run `python setup.py sdist` and then `tar tzvf dist/signingscript-12.1.0.tar.gz`, you'll note that we lose `data/` and `vendored/` from the source tarball. We need the trailing `*` here.

File: requirements/base.txt [hunk 2]
```diff
@@ -21,9 +21,9 @@ aiohttp==3.6.0 \
     --hash=sha256:e0fe698d1e6a852a27a88d2844a1a63839ee764d7cf214fd58cbea480407cc1d \
     --hash=sha256:fa155e309cc2277d6f9d099aecaf3ce78d86a31f5a62a994debc872e4c34ddf4 \
     # via scriptworker, taskcluster
-arrow==0.14.7 \
-    --hash=sha256:4bfacea734ead51495dc47df00421ecfd4ca1f2c0fbe58b9a26eaeddedc31caf \
-    --hash=sha256:67f8be7c0cf420424bc62d8d7dc40b44e4bb2f7b515f9cc2954fb36e35797656
+arrow==0.15.0 \
```
Comment (escapewindow) [inline requirements/base.txt]: Arrow 0.15.0 (may be broken](https://github.com/mozilla-releng/scriptworker/issues/394). We can likely leave it and fix in scriptworker.

File: requirements/test.in [hunk 3]
```diff
@@ -1,11 +1,6 @@
 -r base.in
 
-flake8
-flake8_docstrings
-black
-pydocstyle~=3.0
```
Comment (escapewindow) [inline requirements/test.in]: Are these changes intentional? (Looks like these have been moved into `tox.ini`. Sure, though we lose the ability to update pins)

File: requirements/base.txt (hunk 2 above)
Comment (catlee) [reply requirements/base.txt]: ok, I can revert this.

File: requirements/test.in (hunk 3 above)
Comment (catlee) [reply requirements/test.in]: well, we don't really need them to run the tests...but you're right, we lose pinning by putting them inside tox.

File: requirements/base.txt (hunk 2 above)
Comment (escapewindow) [reply requirements/base.txt]: It looks like we're not affected by it, so feel free to leave it.
